CHANGELOG.md

```diff
@@ -1,17 +1,24 @@
 # UNRELEASED
 
-# v1.27.0
+# v1.28.0
 
 ## Enhancements
-* Adds `RunPreApplyRunning` and `RunQueuingApply` run statuses by @uk1288 [#712](https://github.com/hashicorp/go-tfe/pull/712)
 * Update `Workspaces` to include associated `project` resource by @glennsarti [#714](https://github.com/hashicorp/go-tfe/pull/714)
 * Adds BETA method `Upload` method to `StateVersions` and support for pending state versions by @brandonc [#717](https://github.com/hashicorp/go-tfe/pull/717)
+* Adds support for the query parameter `q` to search `Organization Tags` by name [#720](https://github.com/hashicorp/go-tfe/pull/720)
```

Comment:
Update `name` to the correct username here 😄

Reply:
What do you mean by username?
The Organization Tag attribute is `name` according to the docs [here](https://developer.hashicorp.com/terraform/cloud-docs/api-docs/organization-tags#list-tags)

Reply:
Oh nvm I see what you mean